.github/actions/setup-toolchain/action.yml

```diff
@@ -28,3 +30,19 @@ runs:
     - name: Set up VSTest.console.exe
       if: ${{ inputs.platform == 'windows' }}
       uses: darenm/Setup-VSTest@v1
+    - name: Generate cache key
+      id: cache-key-generator
+      run: |
+        if [[ -f example/${{ inputs.platform }}/Podfile.lock ]]; then
+          echo "::set-output name=cache-key::$(node scripts/shasum.js example/${{ inputs.platform }}/Podfile.lock)"
+        else
+          echo '::set-output name=cache-key::false'
+        fi
+      shell: bash
+    - name: Cache /.ccache
+      if: ${{ steps.cache-key-generator.outputs.cache-key != 'false' }}
+      uses: actions/cache@v2
+      with:
+        path: .ccache
+        key: ${{ runner.os }}-${{ inputs.variant }}-ccache-${{ steps.cache-key-generator.outputs.cache-key }}
+        restore-keys: ${{ runner.os }}-${{ inputs.variant }}-ccache-
```

Comment:
Is something missing after the last '-'? Or is this supposed to be a key prefix for all keys in that could be generated by this run -- e.g. for reuse, or periodic cleanup?

Reply:
The docs aren't 100% clear on this, but I think your assumption is correct: https://github.com/actions/cache#inputs